### DISCUSSION TITLE
Override arguments?

### DISCUSSION THREAD

**Discussion by svrooij** [svrooij/WinTuner#94] [Ideas]
I'm thinking of allowing to override arguments when packaging an app. Some apps (like [SQL management Studio](https://github.com/microsoft/winget-pkgs/blob/957ca3fbac683bc64f79581962b451832ad49a87/manifests/m/Microsoft/SQLServerManagementStudio/20.2/Microsoft.SQLServerManagementStudio.installer.yaml#L7-L13)) don't have the correct silent switches in the manifest, which results in not being able to install it.

Currently working on testing installers in the sandbox, which allows you to quickly test the installer and maybe even figure out how to install it silently.

Then when packaging such an app you would be able to override the arguments, and that way have better packages. Related to https://github.com/svrooij/WingetIntune/issues/50 @jannikk09 what do you think? Is this a good idea?

**Comment by w0** [comment]
Great idea, sandbox stream lines the testing. Any way to save them for future reuse?

**Comment by 4IoTMirko** [comment]
That's a very good idea and pretty important feature for production, too. Many products support customizing via parameters passed to the installer, which would be unlocked via an override function.
